ui/src/components/connections/ConnectionsTable.tsx

```diff
@@ -174,6 +216,12 @@ export function Connections() {
           </Row>
         }
       />
+      {connectionSelected && (
+        <ConnectionDeleteModal
```

Comment:
This (and perhaps the other one too) is missing some sort of refresh for the table, otherwise it may say it's successful but it doesn't show the result.

[Screencast from 05-04-23 12:53:24.webm](https://user-images.githubusercontent.com/2691584/230060638-0a6c2765-4f37-4c58-ba3e-c35b5168d64b.webm)